File: Codeforces/1698/A. XOR Mixup.cpp
#include <bits/stdc++.h>

using namespace std;

void solve() {

    int n, i, arr[105], sum;
    cin >> n;

    sum = 0;

    for (i = 0; i < n; ++i) {
        cin >> arr[i];
        sum ^= arr[i];
        // cout << "sum = " << sum << endl;
    }

    for (i = 0; i < n; ++i) {
        // cout << "sum ^ arr[i]) == arr[i] " <<  (sum ^ arr[i]) << " " << arr[i] << endl;
        if ((sum ^ arr[i]) == arr[i]) {
            cout << arr[i] << endl;
            return;
        }
    }
    return;
}

int main() {

    // freopen("in.txt", "r", stdin);

    int test;

    cin >> test;
    while(test--) {
        solve();
    }

    return (0);
}

File: Codeforces/1698/A. XOR Mixup.go
package main

import (
	"fmt"
)

func solve() {
	var n, i, sum, x int
	var numbers []int

	fmt.Scanf("%d", &n)

	sum = 0

	for i = 0; i < n; i++ {
		fmt.Scanf("%d", &x)
		numbers = append(numbers, x)
		sum ^= x
	}

	for i = 0; i < n; i++ {
		if (sum ^ numbers[i]) == numbers[i] {
			fmt.Printf("%d\n", numbers[i])
			return
		}
	}
}

func main() {

	// os.Stdin, _ := os.OpenFile("in.txt", os.O_RDONLY, 0666)

	var test int
	fmt.Scanf("%d", &test)

	for test > 0 {
		test--
		solve()
	}
}
